.env.example

```diff
@@ -1,3 +1,7 @@
 NODE_ENV=development
 PORT=300
-DATABASE=''
+DATABASE='mongodb://localhost/Fundme'
+
+ACCESS_TOKEN_SECRET = 'bae3241gth6547326gdhgahags7635342625ddhgdss
+jsdvscssxaa5zz6z5sebsrebuyww7varar6vyi62V2734DFe54cdhff'
```

Comment:
Please Remove all these credentials here, its meant to be inside the .env file you created on your own. This is just an example file for the .env file, return this file to the default.